tests/tools/calendar_test.py

```diff
@@ -115,3 +117,66 @@ async def test_calendar_tool_crashes_if_too_many_days(httpd: HTTPServer):
     assert False
   except ValueError as e:
     assert str(e) == "next_days must be at most 366"
+
+class MockHTTPRequestHandler(SimpleHTTPRequestHandler):
```

Comment:
Let's make what it does clear in the class name. Feel free to come up with a shorter name, but, in general, it's better to write a longer name than an unclear one.

```suggestion
class MockHTTPRequestHandlerServingDifferentCalendars(SimpleHTTPRequestHandler):
```